### PR TITLE
[DUOS-2297][risk=no] Update email templates

### PR DESCRIPTION
**Pull request by fboulnois**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-2297

### Summary

Update styling of email templates:

* Use DUOS logo in header
* Remove Broad header
* Switch footer copyright to DUOS
* Use white background
* Switch text to use Montserrat font with same color as UI
* Remove style duplication and proliferation
* Standardize use of `Hello <username>,`
* Standardize "contact us" phrase and ensure it links out to duos support
* Standardize closing phrase to use line break instead of unusual styles
* Use link callout for `new-researcher-library-request.html`
* Center using css instead of `<center>`
* Format html to make it easier to navigate
* Remove invalid uses of `align` and `width`
* Update tests to search for `Hello <username>,`

### Screenshots

Before and after:

01a
01b
02a
02b
03a
03b
04a
04b
05a
05b
06a
06b
07a
07b
08a
08b
09a
09b
10a
10b
11a
11b
12a
12b
13a
13b
14a
14b

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/freemarker/admin-dar-approved.html [hunk 1]
```diff
@@ -1,52 +1,60 @@
...
-    <table width="600" bgcolor="#eeeeee" style="border-collapse: collapse ; font-family: 'Roboto', sans-serif ; box-shadow: 3px 3px 0 #cccccc ; border-radius: 5px ; -moz-border-radius: 5px ; margin-top: 20px; background-color: #eeeeee;">
-        <tr width="600" style="font-family: 'Roboto', sans-serif;">
-            <td align="left" style="font-family: 'Roboto', sans-serif ; font-size: 14px; color: #777777; text-align: left; line-height: 21px; padding: 20px 30px 25px 30px;">
-                <img src="http://imageshack.com/a/img905/7554/gjprR0.png" alt="Broad Institute Logo" style=" margin-top: 10px; display: inline-block; font-family: 'Roboto', sans-serif ;">
+<body style="text-align: center; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; width: 100%; height: 100%; margin: 0; padding: 0;">
+<div style="text-align: left; margin: 0 auto; width: 600px;">
+    <table style="width: 600px">
+        <tr>
+            <td style="padding: 15px 15px 0px; text-align: center;">
+                <img src="https://i.imgur.com/d8rv3c8.png" height="100px" width="100px" alt="DUOS Logo">
```
Comment (fboulnois) [inline src/main/resources/freemarker/admin-dar-approved.html]: Note for reviewers: I will switch the `imgur` links to the logo resource in https://github.com/DataBiosphere/duos-ui/pull/2042 once those assets are in production.

**Review by rushtong — APPROVED** [review]
Nice work, much easier to read 👍🏽

**Review by connorlbark — APPROVED** [review]
👍 looks great, checked all the details of the links, etc., and there seems to be no regressions. great job.

**Review by rushtong — APPROVED** [review]
Changes look very nice! 👍🏽

**Pull request merged**